### ISSUE TITLE
Terminal text all black in Chrome only

### ISSUE THREAD

**Issue by zonywhoop** [cdr/code-server#4090]
## OS/Web Information

- Web Browser: Chrome  92.0.4515.159 and 93.0.4577.63
- Local OS: MacOS 11.5.2
- Remote OS: Ubuntu 18.04
- Remote Architecture: x64
- `code-server --version`: 3.11.1 c680aae973d83583e4a73dc0c422f44021f0140e

## Steps to Reproduce

1. Install and start code-server
2. Connect via Chrome and start a terminal session
3.

## Expected
Terminal output should show all ansi colors correctly.

## Actual
All text is black, regardless of color or bold setting. The same output is shown regardless of shell used and regardless of theme being applied.

## Logs
No errors or exceptions are logged

## Screenshot
Screenshot from Safari 14.1.2
<img width="1130" alt="Screen Shot 2021-09-02 at 7 37 33 AM" src="https://user-images.githubusercontent.com/960482/131845426-d9a5f7a7-288e-4cc6-8023-4b911cc06941.png">

Screenshot from Chrome 93.0.4577.63
<img width="978" alt="Screen Shot 2021-09-02 at 7 37 41 AM" src="https://user-images.githubusercontent.com/960482/131845429-166d05a7-0d73-4140-863f-a174edeceef2.png">

## Notes

This issue can be reproduced in VS Code: No

**Comment by jsjoeio** [comment]
Are you sure this isn't a theme, font or settings issue?

I can't reproduce locally on macOS + Brave

![image](https://user-images.githubusercontent.com/3806031/131893576-977cdbd3-543c-4d8d-95bb-718c5f502f9f.png)

**Comment by zonywhoop** [comment]
Here are some screenshots from the latest version of Brave (1.29.76) on my machine using different color themes.  This only seems to happen on Chromium based browsers.  Safari and Firefox render the terminal fine and the rendering issues do not happen anywhere in in the code-server window that I can tell.

Just as an FYI, this is the first time Brave had loaded this site so there are no cached assets being used here.

Default Light + theme
![Brave - Light Plus](https://user-images.githubusercontent.com/960482/131899505-46417bb4-0247-423c-a9ea-57cc708fbff2.png)

Material Theme
![Brave - Material Theme](https://user-images.githubusercontent.com/960482/131899508-92278bad-3d07-47d8-8ea3-246ecfe33130.png)

Monkai Theme
![Brave - Monokai](https://user-images.githubusercontent.com/960482/131899510-b6a17536-1c03-4ac0-8e2a-9cbcec289ccb.png)

**Comment by jsjoeio** [comment]
Hmm 🤔 This is very very strange. Have you tried disabling all extensions?

Do you mind posting your `settings.json`?

**Comment by zonywhoop** [comment]
I just tried disabling all extensions (command palette -> Extensions: Disable all installed extensions) and then did a refresh. The outcome was the same.  It's definitely very weird.

Here is my `settings.json` file, very basic.
```JSON
{
    "sync.gist": "",
    "sync.autoDownload": false,
    "workbench.colorTheme": "Monokai Dimmed",
    "editor.fontFamily": "'JetBrains Mono', 'Source Code Pro', Hack, Monaco, 'Courier New', monospace",
    "editor.tabSize": 2,
    "editor.fontWeight": 400,
    "terminal.integrated.fontFamily": "'Source Code Pro', 'JetBrains Mono', 'Courier New', Courier",
    "editor.fontSize": 14,
    "files.autoSaveDelay": 5000,
    "terminal.integrated.automationShell.linux": "",
    "terminal.integrated.shell.linux": "/bin/bash",
    "terminal.integrated.shellArgs.linux": [
        "-l"
    ],
    "files.autoSave": "off",
    "yaml.format.enable": true,
    "yaml.format.bracketSpacing": true,
    "workbench.editorAssociations": [
        {
            "viewType": "jupyter-notebook",
            "filenamePattern": "*.ipynb"
        }
    ],
    "redhat.telemetry.enabled": false,
    "terminal.integrated.tabs.enabled": false,
    "workbench.iconTheme": "material-icon-theme"
}
```

**Comment by jsjoeio** [comment]
Bummer!

Can you comment out or remove these lines?
```json
    "workbench.colorTheme": "Monokai Dimmed",
    "editor.fontFamily": "'JetBrains Mono', 'Source Code Pro', Hack, Monaco, 'Courier New', monospace",
    "terminal.integrated.fontFamily": "'Source Code Pro', 'JetBrains Mono', 'Courier New', Courier",
```

On an unrelated note I think
```json
    "workbench.editorAssociations": [
        {
            "viewType": "jupyter-notebook",
            "filenamePattern": "*.ipynb"
        }
    ],
```

causes [this issue](https://github.com/cdr/code-server/issues/3873) and you may want to use [this solution](https://github.com/cdr/code-server/issues/3873#issuecomment-912100324)

**Comment by zonywhoop** [comment]
Update,
I made the change for `workbench.editorAssociations` and that did solve my other issue with the `Unable to write to Folder Settings` notification - so YAY on that!

I also removed all of the above lines regarding theme and fonts and voila the issue was fixed. I then added them back in until it broke again - it was `terminal.integrated.fontFamily`.  I then went through and added the fonts in 1 by 1 and refreshed till it again broke, this time all works until `Source Code Pro` is added in. I'm not sure what about that font is breaking it on Chrome - and only chrome - but that is the issue. It works fine with `JetBrans Mono` or any of the other mono spaced fonts.
